src/app/models/personal.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { IAddress } from './address.interface';
```

Comment:
I've been meaning to look into the use of Interfaces and the reactive forms approach. I don't like that there is so much duplication. It would be really nice if a form could be built from an interface. Just a thought, let me know if you run into anything on the web.